src/components/Modal.js

```diff
@@ -55,7 +55,7 @@ const Modal = ({ onDismiss, title, titleExtras, children, width = 450, showCance
       previouslyFocusedNode.current = modalNode.current.contains(document.activeElement) ? previouslyFocusedNode.current : document.activeElement
       nodeToFocus.focus()
     },
-    style: { overlay: styles.overlay, content: { ...styles.modal, width } },
+    style: { overlay: styles.overlay, content: { ...styles.modal, width, ...props.styles.modal } },
```

Comment:
I checked this out locally and hit an error in this line: `TypeError: Cannot read properties of undefined (reading 'modal')`